src/interfaces/video.ts

```diff
@@ -1,10 +1,16 @@
 import { VideoType } from '../types/video-type'
 export interface Video {
-    description?: string;
+    description: string;
     mediaType: VideoType;
     publishDate: string;
     slug: string;
     thumbnailUrl: string;
     title: string;
     videoId: string;
-}
+    SEO: object;
+    tags: object[];
+    products: object[];
+    languages: object[];
```

Comment:
Curious what you think if we had an interface of `taggable` and put products/languages/tags in there as well to reuse?